### PR TITLE
Update RS and SHOULDMAY config for MS-ASCNTC

### PR DESCRIPTION
**Pull request by XINYU33**
Please review the commit for updating RS and SHOULDMAY config for MS-ASCNTC.

### CODE REVIEW

**Review by XinwLi — APPROVED** [review]

File: ExchangeActiveSync/Source/MS-ASNOTE/TestSuite/MS-ASNOTE_RequirementSpecification.xml [hunk 1]
```diff
@@ -453,12 +633,57 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-ASNOTE_R58</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.2.6</ns1:Doc_Sect>
-		<ns1:Description>[In Subject] The Subject element specifies the subject of the note.</ns1:Description>
+		<ns1:Description>[In MessageClass] The Subject element specifies the subject of the note.</ns1:Description>
```
Comment (v-yahan) [inline ExchangeActiveSync/Source/MS-ASNOTE/TestSuite/MS-ASNOTE_RequirementSpecification.xml]: Why do you change this description?

File: ExchangeActiveSync/Source/MS-ASNOTE/TestSuite/MS-ASNOTE_RequirementSpecification.xml [hunk 2]
```diff
@@ -453,12 +633,57 @@
...
 		<ns1:Behavior>Protocol</ns1:Behavior>
 		<ns1:Scope>Both</ns1:Scope>
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Test Case</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASNOTE_R58001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.6</ns1:Doc_Sect>
```
Comment (v-yahan) [inline ExchangeActiveSync/Source/MS-ASNOTE/TestSuite/MS-ASNOTE_RequirementSpecification.xml]: It should be in section 2.2.2.5

File: ExchangeActiveSync/Source/MS-ASNOTE/TestSuite/MS-ASNOTE_RequirementSpecification.xml [hunk 3]
```diff
@@ -321,7 +321,52 @@
...
 		<ns1:Behavior>Protocol</ns1:Behavior>
 		<ns1:Scope>Both</ns1:Scope>
-		<ns1:IsNormative>Informative</ns1:IsNormative>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
+	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASNOTE_R43001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.1</ns1:Doc_Sect>
+		<ns1:Description>[In Body] Protocol version 12.1 does not supports this [In Body] element.</ns1:Description>
```
Comment (v-yahan) [inline ExchangeActiveSync/Source/MS-ASNOTE/TestSuite/MS-ASNOTE_RequirementSpecification.xml]: "this" indicates "Body" but not "In Body". So please update it to:
[In Body] Protocol version 12.1 does not supports this [Body] element.
Please also update the similar issues.

**Review by v-yahan — COMMENTED** [review]
Please add MS-ASNOTE.md file.

**Review by v-yahan — APPROVED** [review]

**Pull request merged**